src/Game.cpp

```diff
@@ -196,6 +374,27 @@ void sub_6134D0(const char *var8) {
     }
 }
 
+bool sub_612EB0(const char * key, std::uint32_t value_index, const char ** value_arg) {
+    unk_612150 *value = glo_8600F8.sub_6124A0(key);
+
+    if (value == nullptr)
+        return false;
+
+    if (value_index >= value->count)
+        return false;
+
+    *value_arg = value->args[value_index];
+    return true;
+}
+
+void sub_5ECBE0() {
+    if (glo_6754C4 != MC2_PROC_PTR<void>(0x005ECB90)) {
```

Comment:
I don't like the idea of using `MC2_PROC_PTR` in this way. This *should* be `&sub_5ECB90` to fit the standard I want, but that doesn't work based on how I've been defining `MC2_PROC_PTR`. For now, this should probably be `if (glo_6754C4 != sub_5ECB90)`, and I'll work on updating how the function pointer stuff works (It should probably be a function reference).